structure/0001-rfc-process.md

```diff
@@ -46,22 +46,18 @@ When you are ready to submit an RFC, the process will work as follows:
 - The team responsible for the RFC will discuss the RFC as much as possible in
   the pull request's comments. Discussion that occurs in other spaces should be
   summarized in the PR comments.
-- When the team responsible for the RFC decides that there has been enough discussion, a member of the team will issue a "motion for final comments". If the motion passes, it begins a 10 day period where the community has the last chance to give feedback on an RFC.
+- When the team responsible for the RFC decides that there has been enough discussion, a leader of the team will issue a "motion for final comments". If the motion passes, it begins a short period where the community has the last chance to give feedback on an RFC. This period must last at least 3 days and cannot last more than 7 days.
   + This step is taken when enough discussion has happened to make a final decision on the issue. It is designed to prevent RFCs from becoming stales.
-  + A consensus among all community members *is not necessary* for an RFC to
+  + A consensus among all community members *is not necessary nor expected* for an RFC to
     enter the final comment period. However, there should not be a strong
     consensus *against* the disposition given for the final comment period, and
```

Comment:
This still references disposition which have been removed as a requirement, should this be made more general e.g. there shouldn't be a strong consensus against the PR before an FCP can take place.